staging/src/k8s.io/api/roundtrip_test.go

```diff
@@ -1,5 +1,6 @@
 /*
 Copyright 2018 The Kubernetes Authors.
+Copyright 2020 Authors of Arktos - file modified.
```

Comment:
nit: copyright should be Kubernetes only

Reply:
Just talked in scalability meeting, that is OK to add arktos copyrights.

Reply:
Added cherrypick comments for all added/updated files